conf.d/python.d/isc_dhcpd.conf

```diff
@@ -75,4 +78,6 @@
 #
 #leases:
 #  leases_path : '/var/lib/dhcp/dhcpd.leases'
-#  pools       : '192.168.3.0/24 192.168.4.0/24 192.168.5.0/24'
+#    pools:
```

Comment:
I think the spacing here is wrong. `pools` should be aligned with `leases_path`.
Yes, it commented, but uncommenting it and running it gives error.